crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -22,7 +24,8 @@ pub struct PowertranzPaymentsRequest {
     three_d_secure: bool,
     source: Source,
     order_identifier: String,
-    billing_address: Option<PowertranzAddressDetails>,
+    // billing_address: Option<PowertranzAddressDetails>,
```

Comment:
remove commented lines

Reply:
This is intentionally commented, these are optional fields and expects ISO code for state field